File: 04/StudentManagement.java
import java.util.*;

public class StudentManagement {
    public static double truncate(float val){
        int temp = (int)(val*100);
        double result = (double)temp/100;
        return result;
    }
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        String name = sc.nextLine();
        int age = sc.nextInt();
        float cgpa = sc.nextFloat();
        char grade = sc.next().charAt(0);

        System.out.printf("""
                Name: %s
                Age: %d
                CGPA: %.2f
                Grade: %c
                """, name, age, truncate(cgpa), grade);

        sc.close();
    }
}

File: 04/StudentManagement.py

def main():
    name = input().strip()
    age = int(input())
    cgpa = float(input())
    grade = input()

    print(
        f"""
        Name: {name}
        Age: {age}
        CGPA: {truncate(cgpa)}
        Grade: {grade}
        """
    )

def truncate(cgpa: float) -> float:
    temp = int(cgpa*100)
    result = float(temp)/100
    return result

if __name__=="__main__":
    main()
